### PR TITLE
 Accuracy Checker: provide latest changes 

### PR DESCRIPTION
**Pull request by eaidova**
Changes:
* supported device extraction from config and providing full paths for runtime specific files in command line (e. g. bitstream, affinity map)
* enabled multi device plugin for DLSDK launcher
* separated preprocessors to different modules (pylint module size restriction)
* returned back previously used matching approach for detection accuracy
* extended support MS COCO original metrics (added way to calculate metrics for keypoints and instance segmentation task using pycocotools)
* supported new modeles for text detection workflow: CTPN, [EAST](https://github.com/argman/EAST)
* added opportunity to overwrite model inputs using configs for TF launcher

### CODE REVIEW

File: tools/accuracy_checker/accuracy_checker/adapters/README.md [hunk 1]
```diff
@@ -82,9 +82,30 @@ AccuracyChecker supports following set of adapters:
 * `super_resolution` - converting output of single image super resolution network to `SuperResolutionPrediction`.
   * `reverse_channels` - allow switching output image channels e.g. RGB to BGR (Optional. Default value is False).
 * `landmarks_regression` - converting output of model for landmarks regression to `FacialLandmarksPrediction`.
-* `text_detection` - converting output of model for text detection to `TextDetectionPrediction`.
+* `pixel_link_text_detection` - converting output of PixelLink like model for text detection to `TextDetectionPrediction`.
   * `pixel_class_out` - name of layer containing information related to text/no-text classification for each pixel.
   * `pixel_link_out` - name of layer containing information related to linkage between pixels and their neighbors.
+  * `pixel_class_confidene_threshold` - confidence threshold for valid segmentation mask (Optional, default 0.8).
```
Comment (kchechil) [inline tools/accuracy_checker/accuracy_checker/adapters/README.md]: confidene=>confidence?

Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/adapters/README.md]: Fixed, thanks!

File: tools/accuracy_checker/configs/text-detection-0003.yml [hunk 2]
```diff
@@ -106,6 +70,6 @@ models:
             apply_to: prediction
 
         metrics:
-          - type: text_detection
+          - type: incedental_text_hmean
```
Comment (IRDonch) [inline tools/accuracy_checker/configs/text-detection-0003.yml]: ITYM "incidental"

File: tools/accuracy_checker/accuracy_checker/adapters/action_recognition.py [hunk 3]
```diff
@@ -1,6 +1,5 @@
 """
 Copyright (c) 2019 Intel Corporation
-
```
Comment (IRDonch) [inline tools/accuracy_checker/accuracy_checker/adapters/action_recognition.py]: This seems wrong. The otherfiles do have this blank line.

File: tools/accuracy_checker/configs/text-detection-0003.yml (hunk 2 above)
Comment (eaidova) [reply tools/accuracy_checker/configs/text-detection-0003.yml]: Fixed

File: tools/accuracy_checker/accuracy_checker/adapters/action_recognition.py (hunk 3 above)
Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/adapters/action_recognition.py]: Fixed

**Pull request merged**